app/src/utils/utils.ts

```diff
@@ -89,7 +89,16 @@ export const stringToSlug = (string: string = ""): string => {
 };
 
 export const titleToDCParam = (string: string = ""): string => {
-  return string?.replace(/\s+/g, "+"); // replace spaces with +
+  // replace space with + and & with '%26'
+  return string?.replace(/(\s+)|(&)/g, (match, whitespace, ampersand) => {
+    if (ampersand) {
+      return "%26";
+    }
+    if (whitespace) {
+      return "+";
+    }
+    return match;
+  });
```

Comment:
this works, but I wonder if we could just use the JS builtin [encodeURIComponent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent)?